bugbug/similarity.py

```diff
@@ -97,6 +101,10 @@ def __init__(
 
         self.nltk_tokenizer = nltk_tokenizer
         self.confidence_threshold = confidence_threshold
+        self.end_to_end = end_to_end
```

Comment:
Instead of storing this, just do:
```
self.duplicate_model = DuplicateModel.load("duplicatemodel") if end_to_end else None
```

And in `get_similar_bugs` check `if self.duplicate_model is not None` instead of `if self.end_to_end`.